api/v1_coins_volume_leaders.go

```diff
@@ -0,0 +1,168 @@
+package api
+
+import (
+	"time"
+
+	"api.audius.co/api/dbv1"
+	"github.com/gofiber/fiber/v2"
+	"github.com/jackc/pgx/v5"
+)
+
+type VolumeLeaderRow struct {
+	Address string  `json:"address" db:"address"`
+	Volume  float64 `json:"volume" db:"volume"`
+	UserID  *int32  `json:"-" db:"user_id"`
+}
+
+type VolumeLeaderUserFull struct {
+	VolumeLeaderRow
+	User *dbv1.FullUser `json:"user"`
+}
+
+type VolumeLeaderUserMin struct {
+	VolumeLeaderRow
+	User *dbv1.MinUser `json:"user"`
```

Comment:
Do we need whole user objects for the UI? Also are we giving up on the separate "related_entities" top level thing?

Reply:
We don't need it, but I chose to do it for:
* Keeping SDK responses consistent across endpoints (because it's called `user`)
* If we do decide to call this somewhere in the app, the field won't be a partial user record, so it's cacheable.

I agree that I don't love embedding user. Happy to move it up to related entities. I don't think I had full buy in on using that pattern, so I was going with the existing convention.

Reply:
yeah that makes sense, probably good to keep this way